cluster/cluster.go

```diff
@@ -165,8 +170,8 @@ func (c *Cluster) DeployWorkerPlane(ctx context.Context, svcOptionData map[strin
 		workerNodePlanMap[workerHost.Address] = BuildRKEConfigNodePlan(ctx, c, workerHost, workerHost.DockerInfo, svcOptionData)
```

Comment:
Can we change this loop from `for _, workerHost` to `for _, host`, as we define what it is later by checking what role(s) it has and it's confusing to call it `workerHost` from the beginning. (I know this is not related to this PR specifically)